### PR TITLE
Add support for `service_external_ips_config` to GKE cluster modules

### PR DESCRIPTION
**Pull request by luigi-bitonti**
Variable `service_external_ips_config` was added to GKE Autopilot and Standard module.
---
**Checklist**

I applicable, I acknowledge that I have:
- [x] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [x] Ran `terraform fmt` on all modified files
- [x] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [x] Made sure all relevant tests pass

### CODE REVIEW

**Review by juliocc — CHANGES_REQUESTED** [review]
instead a of a new variable, can you add a new field to `var.eanble_features`?

File: modules/gke-cluster-standard/main.tf [hunk 1]
```diff
@@ -110,6 +110,9 @@ resource "google_container_cluster" "cluster" {
       enabled = var.backup_configs.enable_backup_agent
     }
   }
+  service_external_ips_config {
```
Comment (juliocc) [inline modules/gke-cluster-standard/main.tf:113]: Make the block dynamic like (see the surrounding blocks)

**Review by juliocc — CHANGES_REQUESTED** [review]

**Review by ludoo — APPROVED** [review]

**Comment by juliocc** [comment]
@luigi-bitonti I'm closing this. Feel free to reopen if you want to address the comments above

**Comment by luigi-bitonti** [comment]
> @luigi-bitonti I'm closing this. Feel free to reopen if you want to address the comments above

Hi Julio, I opened this PR with the changes you requested (dynamic block): https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/pull/1970

**Pull request closed without merging**